src/styles/styles.js

```diff
@@ -1425,11 +1425,16 @@ const styles = {
     },
 
     textInputComposeSpacing: {
-        paddingVertical: 6,
+        alignSelf: "center",
         ...flex.flexRow,
         flex: 1,
     },
```

Comment:
Why deviation from the proposed changes?

@Expensify/design Are we fine removing `paddingVertical` and using `alignSelf: "center"`

cc: @Julesssss

Reply:
I don't feel strongly so long as we stress test this across platforms and it behaves as expected. Curious what Jules thinks though.

Reply:
@Santhosh-Sellavel this is the solution which resolves the alignment for **all** platforms.

Reply:
Yeah it would have been better to realise this during the proposal stage. But the change seems okay to me 👍

Reply:
> Yeah it would have been better to realise this during the proposal stage. But the change seems okay to me

Agree, but it was working fine then, some recent changes might break something. I remember testing this across all platforms.